<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_DialogDatabaseRunQuery</class>
 <widget class="QDialog" name="Dialog_DialogDatabaseRunQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>201</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Run Query</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QDialogButtonBox" name="button_DialogDatabaseRunQuery">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_DialogDatabaseRunQuery_RemoveData">
       <property name="text">
        <string>Clear Database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_IssueToItem">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Link all word order data to respective Item tags?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_TagNg">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Store the Multi-Word data that you have tagged ?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_OriginalData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Store the data from your Work Order CSV ?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_Tag1g">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Store the Single Word data that you have tagged ?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_ItemToItem">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Link Item to Item hierarchy ?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_DialogDatabaseRunQuery_1gToNg">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Link all Single Words to respective Mulit-Word ?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_DialogDatabaseRunQuery</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_DialogDatabaseRunQuery_OriginalData</tabstop>
  <tabstop>checkBox_DialogDatabaseRunQuery_Tag1g</tabstop>
  <tabstop>checkBox_DialogDatabaseRunQuery_TagNg</tabstop>
  <tabstop>checkBox_DialogDatabaseRunQuery_1gToNg</tabstop>
  <tabstop>checkBox_DialogDatabaseRunQuery_IssueToItem</tabstop>
  <tabstop>checkBox_DialogDatabaseRunQuery_ItemToItem</tabstop>
  <tabstop>pushButton_DialogDatabaseRunQuery_RemoveData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_DialogDatabaseRunQuery">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
